python/activator/activator.py

```diff
@@ -108,7 +113,9 @@ def check_for_snap(
         was found. If multiple files match, this function logs an error
         but returns one of the files anyway.
     """
-    prefix = f"{instrument}/{detector}/{group}/{snap}/"
+    prefix = get_prefix_from_visit(instrument, group, detector, snap)
+    if not prefix:
+        return None
```

Comment:
I'm not following. It looks to me that the LSST cases get `None` prefix. Then they all return here?

Reply:
Yes, the LSST cases do not do any looking for previous files at this point.  I think that history mechanisms in Kafka will mean that this happens very rarely (only on container cold start, if that).  But I also have plans to implement an additional microservice that will allow lookup of images from visit/snap information.  It won't happen very soon, though, so this was thought to be good enough for now.